src/providers/arcgisrest/qgsarcgisrestdataitemguiprovider.cpp

```diff
@@ -142,21 +160,21 @@ void QgsAfsDataItemGuiProvider::deleteConnection( QgsDataItem *item )
     item->parent()->refreshConnections();
 }
 
-void QgsAfsDataItemGuiProvider::refreshConnection( QgsDataItem *item )
+void QgsArcGisRestDataItemGuiProvider::refreshConnection( QgsDataItem *item )
 {
   item->refresh();
   // the parent should be updated
   if ( item->parent() )
     item->parent()->refreshConnections();
 }
 
-void QgsAfsDataItemGuiProvider::saveConnections()
+void QgsArcGisRestDataItemGuiProvider::saveConnections()
 {
   QgsManageConnectionsDialog dlg( nullptr, QgsManageConnectionsDialog::Export, QgsManageConnectionsDialog::ArcgisFeatureServer );
   dlg.exec();
 }
 
-void QgsAfsDataItemGuiProvider::loadConnections( QgsDataItem *item )
+void QgsArcGisRestDataItemGuiProvider::loadConnections( QgsDataItem *item )
```

Comment:
Do we need to add code to handle pre 3.18 saved connections that'd include ArcgisMapServer?

Reply:
That's done here: https://github.com/qgis/QGIS/pull/40700/files#diff-5fab4f0ef4a0231dcfa1488222c365a5955f1df79cfaf06e4086275ea6bd542bR611